feat: header blocklist (#2)
pactflow/cypress-pact: README.md

import 'package-name'
```

## Configuration
By default, this plugin omits most cypress auto-generated HTTP headers. 

> To exclude more customised headers in your pact, add allowed headers as a list of string in `cypress.json` under key `env.headersBlocklist`. Eg. in your `cypress.json`

Reviewer: ```suggestion
To exclude other headers in your pact, add them as a list of strings in `cypress.json` under key `env.headersBlocklist`. Eg. in your `cypress.json`
```

Reviewer 2: 👍🏾 

suggestion - add them as an array of strings

```js
{
    ...otherCypressConfig,
    "env": {
        'headersBlocklist': ['ignore-me-globally']
    }
}
```

> Note: Header blocklist can be set up at test level. Check command (cy.setupPactHeaderBlocklist)[/#cy.setupPactHeaderBlocklist([headers])]

Reviewer 2: The `[]` and `()` order just needs switching here

`[cy.setupPactHeaderBlocklist](/#cy.setupPactHeaderBlocklist([headers]))`
